.gitattributes

```
@@ -0,0 +1 @@
+*.gltf -diff
```

Apparently this actually needed to be 
```suggestion
*.gltf binary
```

I misread the docs